### PR TITLE
Blob is 20 required players

### PR DESCRIPTION
**Pull request by ghost**
We never get to play blob on this server nobody fucking readies ;_;

### CODE REVIEW

**Comment by ghost** [comment]
Lobby timer is now 3 minutes because wow nobody readies (including me)

Like shit I can make a poll on this if you'd like but holy shit every time I come back if I go get water or something the round has already started with a prob of traitor only and it sucks

**Comment by Atskadan** [comment]
no this is really good

**Comment by GunHog** [comment]
MERGE NOW PLZ :+1:

**Comment by Iamgoofball** [comment]
uhm

the lobby timer should of been in another pr

but okay

**Comment by GunHog** [comment]
Sounds like a config thing. SoS could probably change it on the fly. I am fine with it, but I do not think one minute or ten would make people ready up.

**Comment by Iamgoofball** [comment]
I don't, but it probably should of been in a separate PR if someone wanted to find when this was added and what PR it was in. They'll just see "oh this only touches blob, it isn't this one" because of the title

**Comment by GunHog** [comment]
Seems an easy issue to fix. Rename the PR!

**Comment by Atskadan** [comment]
youre one to talk about hiding things in pr's

**Comment by Iamgoofball** [comment]
Honk

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 /datum/configuration
 	var/server_name = null				// server name (for world name / status)
 	var/server_suffix = 0				// generate numeric suffix based on server port
-	var/lobby_countdown = 120			// In between round countdown.
+	var/lobby_countdown = 180			// In between round countdown.
```
Comment (Giacom) [inline code/controllers/configuration.dm:11]: You need to change the config setting itself too, in the .txt. You will then have to tell SoS about the change so he can change it on Artyom.

**Pull request merged**